src/main/java/online/carsharing/service/impl/NotificationServiceImpl.java

```diff
@@ -129,6 +114,16 @@ public void createRentalNotification(Rental rental) {
                 carRepository.modelCheckById(rental.getCar().getId()),
                 email
         );
+        sendMessageToAllChats(messageText);
+    }
+
+    @Override
+    public void sendNotification(String message) {
+        sendMessageToAllChats(message);
+    }
+
+    private void sendMessageToAllChats(String messageText) {
+        List<Long> chatIds = userChatIdRepository.findAllChatIds();

```

Comment:
think about it, maybe it make sense:
```suggestion
        List<Long> chatIds = chatRepository.findAll();

```

Reply:
Method findAllChatId should stay because of changs to table
chaRepository - ok I can do that